lib/galaxy/datatypes/sniff.py

```diff
@@ -253,75 +254,75 @@ def is_column_based( fname, sep='\t', skip=0, is_multi_byte=False ):
     return True
 
 
-def guess_ext( fname, sniff_order=None, is_multi_byte=False ):
+def guess_ext( fname, sniff_order, is_multi_byte=False ):
```

Comment:
:scream_cat: How did none of the tests catch that, I'll look into this - thanks @nsoranzo.